stories/Components/UIcomponents/Gallery/LightboxGallery/lightbox.scss

```diff
@@ -1,13 +1,28 @@
 @import "../../../../assets/scss/variables";
 @import "../../../../assets/scss/mixins";
+@import "../../../../assets/scss/breakpoints";
 
+// backgroundoverlay
+.goverlay {
```

Comment:
@harishkumar-srijan Kindly change the class name to `glightbox`.

Reply:
goverlay is default class of glightbox, so we can't change it.